#goal
int32 objectType
int32 diceNum
bool servoing

#Constants for object types
int32 startGate=0
int32 pole=1
int32 dice=2
int32 dice1=3
int32 dice2=4
int32 dice3=5
int32 dice4=6
int32 dice5=7
int32 dice6=8
int32 path=9
---
#result
bool stoppedOk
bool found
bool poleDone
---
#feedback
int32[] center
int32 width
int32 height
bool found

#Pose used to pass back fix information
#If fix type is direction, the point component is a direction vector
#If fix type is position, point component is the position in camera space (add header for frame id?)
#if fix type is pose, point and quaternion are pose in camera space
#quaternion is only valid for pose fix type
geometry_msgs/Pose targetPose
